### ISSUE TITLE
Enhancement: Remove the QueryBuilder on the IStatementBuilder interface methods

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDb#1055]
### Describe the enhancement

To simplify the calls to the [IStatementBuilder](https://github.com/mikependon/RepoDB/blob/master/RepoDb.Core/RepoDb/Interfaces/IStatementBuilder.cs) methods without requiring the caller to explicitly instantiate the [QueryBuilder](https://github.com/mikependon/RepoDB/blob/master/RepoDb.Core/RepoDb/QueryBuilder.cs) class, we therefore proposing to remove this argument in all the methods of the Statement Builders.

Current implementation is below.

```csharp
public virtual string CreateQuery(QueryBuilder queryBuilder,
            string tableName,
            IEnumerable<Field> fields,
            QueryGroup where = null,
            IEnumerable<OrderField> orderBy = null,
            int? top = null,
            string hints = null)
```

Proposal impelentation is below.

```csharp
public virtual string CreateQuery(string tableName,
            IEnumerable<Field> fields,
            QueryGroup where = null,
            IEnumerable<OrderField> orderBy = null,
            int? top = null,
            string hints = null)
```

### Additional Context

All the extended libraries (i.e: RepoDb.SqlServer, RepoDb.PostgreSql, RepoDb.MySql, RepoDb.SqLite) must also follow the structure.

### Acceptance Criteria

All the existing Unit Tests and Integration Tests must be updated to conform with the proposal and must be executed successfully.

**Comment by mikependon** [comment]
Related: https://github.com/mikependon/RepoDB/issues/1036

**Comment by mikependon** [comment]
This will be available in the releases > RepoDB v1.12.10.
